### PR TITLE
scvi in R tutorial

### PR DESCRIPTION
**Pull request by galenxing**
still trying to get output in the notebook. its automatically going to terminal.

### CODE REVIEW

File: scvi_in_R.ipynb [hunk 1]
```diff
@@ -0,0 +1,731 @@
+{
```
Comment (adamgayoso) [inline scvi_in_R.ipynb:1]: <p>install instructions should describe conda as the easiest solution. mention pytorch for gpu etc, link to our install.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YosefLab/scvi-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (adamgayoso) [inline scvi_in_R.ipynb:1]: <p>not sure the last line does anything here, shouldn't it be scvi$settings$progress_bar_style = "tqdm"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YosefLab/scvi-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (adamgayoso) [inline scvi_in_R.ipynb:1]: <p>let's add counts to the getassaydata call https://www.rdocumentation.org/packages/Seurat/versions/3.1.4/topics/GetAssayData</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YosefLab/scvi-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (adamgayoso) [inline scvi_in_R.ipynb:1]: <p>n_epochs is automatic now</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YosefLab/scvi-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (adamgayoso) [inline scvi_in_R.ipynb:1]: <p>add counts to getassaydata</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YosefLab/scvi-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (adamgayoso) [inline scvi_in_R.ipynb:1]: <p>feature plot in a new cell, all the umap plots with expression should be the lognormalized data, so you will have to run lognormalization in seurat, which is why you would use the counts option in getassaydata</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YosefLab/scvi-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (adamgayoso) [inline scvi_in_R.ipynb:1]: <p>should use the lognormalized data</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YosefLab/scvi-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (galenxing) [reply scvi_in_R.ipynb:1]: <p>Yea, I wanted to show how to actually set it though since you need to do this as.integer thing. </p>

**Pull request merged**